api/v3/examples/Setting/GetFields.ex.php

```diff
@@ -1774,21 +1774,6 @@ function setting_getfields_expectedresult() {
         'description' => '',
         'help_text' => '',
       ],
-      'moneyvalueformat' => [
```

Comment:
I'm thinking this may cause a test failure but we will see

Reply:
It's in the example - I left the actual metadata

Reply:
I have seen some unit tests refer to the examples as the expected output but we will see